### PR TITLE
E2e kind

### PR DESCRIPTION
**Pull request by coleenquadros**

### CODE REVIEW

**Comment by coleenquadros** [comment]
[
Screenshot 2024-04-11 at 14 52 56
](url)

File: operators/endpointmetrics/manifests/prometheus/kube-state-metrics-clusterRoleBinding.yaml [hunk 1]
```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: kube-state-metrics
-  namespace: open-cluster-management-addon-observability
+  namespace: open-cluster-management-observability
```
Comment (philipgough) [inline operators/endpointmetrics/manifests/prometheus/kube-state-metrics-clusterRoleBinding.yaml:12]: is it ok to hardcode change all these now or is it just for the purposes of getting the tests passed?

**Pull request closed without merging**